### PR TITLE
SEMPS-3334 Update 2FA docs

### PR DESCRIPTION
**Pull request by willllewellyn**
Updated the 2FA reset and setup docs

### CODE REVIEW

**Comment by suehighmoor** [comment]
Can we use a word other than "seed"? In the dialog box it says secret key, so it make sense to me to use the same terminology as that will make it clear to the user what they need to copy.
Also, should the secret key in the images is not blurred out? I know it doesn't show the full key, but it might be better to blur it anyway.

**Comment by willllewellyn** [comment]
> Can we use a word other than "seed"? In the dialog box it says secret key, so it make sense to me to use the same terminology as that will make it clear to the user what they need to copy.
> Also, should the secret key in the images is not blurred out? I know it doesn't show the full key, but it might be better to blur it anyway.

Blured the images, blanked the secret key and changed seed to secret key

File: articles/portal/ptl-how-reset-2fa.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+If a user loses or replaces the device where their authenticator app is installed, they will be unable to generate the required code and therefore be unable to log in to the Portal. In this situation, an administrator must reset the user's 2FA so that they can set up 2FA on a different device.
+
+### Intended audience
+
+This article is intended for Portal administrators who need to reset a Portal user's 2FA.
+
+## Resetting 2FA for a user account
+
+1. Log in to the UKCloud Portal as a Portal administrator.
```
Comment (willllewellyn) [inline articles/portal/ptl-how-reset-2fa.md]: Is it worth adding that the Admin must also have 2FA enabled for them to see the reset button?

Comment (suehighmoor) [reply articles/portal/ptl-how-reset-2fa.md]: Done

**Pull request merged**